Assets/Scripts/Data Model/RasterLayer.cs

```diff
@@ -54,8 +57,8 @@ public RasterLayer(LayerMeta layerMeta/*, PlanLayer planLayer = null*/)
 	{
 		entityTypesSortedByValue = new List<EntityType>(EntityTypes.Values);
 		entityTypesSortedByValue.Sort(SortMethodEntityTypesByValue);
-
 		viewingRaster = rasterAtLatestTime;
+		rasterValueToEntityValueMultiplier = layerMeta.layer_entity_value_max ?? DEFAULT_RASTER_VALUE_TO_ENTITY_VALUE_MULTIPLIER;
```

Comment:
added the possibility to set the maximum value for an layer entity type.
E.g. if this is set to 255 , its wil match possible raw raster values, which Magali likes when using point interpolation mode
Its a per layer setting, so when using color interpolation mode, you can still use a higher value. 
If not set, it will still use the older, and default value 1000